tests/unit/endActivityFunction.unitTest.ts

```diff
@@ -3,16 +3,16 @@ import {ActivityService} from "../../src/services/ActivityService";
 import mockContext from "aws-lambda-mock-context";
 import {HTTPResponse} from "../../src/utils/HTTPResponse";
 
-describe("startActivity Function", () => {
+describe("endActivity Function", () => {
   context("calls activity service", () => {
-    const ctx  = mockContext();
+    const ctx = mockContext();
     context("gets a successful response", () => {
-      it("returns 201 and stringinfied empty string as body", async () => {
-        ActivityService.prototype.endActivity = jest.fn().mockResolvedValue(1234);
-        const resp: HTTPResponse = await endActivity({pathParameters: {id: 1}}, ctx, () => { return; });
+      it("returns 200 containing the response body value", async () => {
+        ActivityService.prototype.endActivity = jest.fn().mockResolvedValue({ visitClosed: false });
+        const resp: HTTPResponse = await endActivity({ pathParameters: { id: 1 } }, ctx, () => { return; });
         expect(resp).toBeInstanceOf(HTTPResponse);
-        expect(resp.statusCode).toEqual(204);
-        expect(resp.body).toEqual(JSON.stringify(""));
+        expect(resp.statusCode).toEqual(200);
+        expect(resp.body).toEqual(JSON.stringify({ visitClosed: false }));
```

Comment:
This doesn't look right.  For a start its using the wrong attribute name - visitClosed was changed to wasVisitAlreadyClosed.  Also as this test passes, its effectively just checking that the mocked value was mocked correctly.  What is it trying to do?